### DISCUSSION TITLE
SDK 'DotNet.ReproducibleBuilds.Isolated' not be found when opening solution

### DISCUSSION THREAD

**Discussion by WimLeflere** [oxyplot/oxyplot#2106] [Q&A] [answered]
None of the projects in the solution(s) load because of following error
`oxyplot\Source\OxyPlot\OxyPlot.csproj : error  : The SDK 'DotNet.ReproducibleBuilds.Isolated/1.1.1' specified could not be found`
How do I install this dependency?
Or where can I find the documentation on setting up the development environment?

**Comment by VisualMelon** [comment]
1.1.1 is still listed, so shouldn't be any issue retrieving it from NuGet: I'd suggest checking your nuget sources to make sure they include NuGet.org is enabled.

There shouldn't be any set up required, but visual studio gets upset sometimes and `dotnet restore`ing and `build`ing the relevant sln from the command line usually sorts it out.

**Reply by WimLeflere** [reply, marked as answer]
NuGet.org was available as source, but wasn't included in the package source mapping (Nuget.config in parent folder).

I moved the repository to a folder without package source mapping and the build now works.

Thanks
